apps/love-producer/values-tucson-teststand.yaml

```diff
@@ -102,6 +105,9 @@ producers:
   dsm2:
     name_index: DSM:2 --log-level 10
     WEBSOCKET_HOST: love-nginx-service/love/manager/producers/ws/subscription
+  electrometer:
```

Comment:
Which index is this?

Reply:
I now understand the intention is to mirror the summit configuration indexes. So we will have three indices: 101, 102 and 201.